wallet/wallet.c

```diff
@@ -3094,7 +3096,13 @@ struct amount_msat wallet_total_forward_fees(struct wallet *w)
 	res = db_step(stmt);
 	assert(res);
 
-	db_column_amount_msat(stmt, 0, &total);
+	if (db_column_is_null(stmt, 0))
+		/* Some databases return a NULL result if there are no
+		 * operands for the SUM(). This is the case for example for
+		 * postgres. */
+		total = AMOUNT_MSAT(0);
```

Comment:
`COALESCE(SUM(in_msatoshi - out_msatoshi), 0)` is Sequelly I think?

Reply:
Yeah, problem is `COALESCE` doesn't result in a `BIGINT`, so a `CAST(COALESCE(field, 0) AS BIGINT)` is required.